packages/compass-saved-aggregations-queries/src/components/saved-item-card.tsx

```diff
@@ -19,7 +19,7 @@ import {
 import type { Item } from '../stores/aggregations-queries-items';
 import { formatDate } from '../utlis/format-date';
 
-export type Action = 'open' | 'delete' | 'copy' | 'rename';
+export type Action = 'open' | 'delete' | 'copy' | 'edit';
```

Comment:
I'd double-check the name with product just in case. IIRC we wanted it to be "Rename" because "Edit" reads as if you can edit the whole thing, but you can only rename it